Pages/js/first_run.js

```diff
@@ -8,6 +8,48 @@
  * Initialize the applications by showing and hiding the proper
  * elements.
  */
+var callbacks = {
+
+  /**
+   * Submit the registration form and await the return of the registration.
+   */
+  submitRegistration: function() {
+    $("#registerModal").modal('hide');
+    privlyNetworkService.sameOriginPostRequest(
+      privlyNetworkService.contentServerDomain() + "/users/invitation",
+      callbacks.checkRegistration,
+      { "user[email]":  $("#register_email").val() });
+  },
+
+  /**
+   * Check to see if the user's registration was accepted by the server.
+   */
+  checkRegistration: function(response) {
+     if ( response.json.success === true ) {
+      callbacks.pendingRegistration();
+    } else {
+      callbacks.registrationFailure();
+    }
+  },
+
+  /**
+   * Tell the user their registration was submitted.
+   */
+  pendingRegistration: function() {
+    $("#messages").html("<strong>Thanks!</strong> If your email isn't already in our " +
+      "database you should receive an email shortly.");
+    $("#messages").show();
```

Comment:
@smcgregor This message shows at the top of the screen after submitting the email.  Did you miss it? If so it should probably change to be more obvious.